src/Commands/DrushCommands.php

```diff
@@ -18,17 +18,20 @@
 use Robo\Contract\ConfigAwareInterface;
 use Robo\Contract\IOAwareInterface;
 use Robo\Common\IO;
+use Robo\LoadAllTasks;
+use Robo\Contract\BuilderAwareInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Consolidation\SiteProcess\ProcessManagerAwareTrait;
 use Consolidation\SiteProcess\ProcessManagerAwareInterface;
 use Webmozart\PathUtil\Path;
 
-abstract class DrushCommands implements IOAwareInterface, LoggerAwareInterface, ConfigAwareInterface, ProcessManagerAwareInterface
+abstract class DrushCommands implements IOAwareInterface, LoggerAwareInterface, ConfigAwareInterface, ProcessManagerAwareInterface, BuilderAwareInterface
```

Comment:
I did wonder if adding BuilderAwareInterface here might break existing commands already using Robo tasks, but it does work in PHP 7.4 and 8.1 to declare BuilderAwareInterface and LoadAllTasks both here and in the implementing Drush command class.